src/cpp/session/modules/environment/SessionEnvironment.cpp

```diff
@@ -1291,7 +1291,7 @@ Error initialize()
       (bind(registerRpcMethod, "remove_objects", removeObjects))
       (bind(registerRpcMethod, "remove_all_objects", removeAllObjects))
       (bind(registerRpcMethod, "get_environment_state", getEnv))
-      (bind(registerRpcMethod, "get_object_contents", getObjectContents))
+      (bind(registerRpcMethod, "getObjectect_contents", getObjectContents))
```

Comment:
Is this really intentional?

Reply:
Definitely doesn't look like it - I'd blame it and discuss with the owner.

Reply:
@MariaSemple it feels like an accidental search replace done in Pro.
The matching code seems to be:
https://github.com/rstudio/rstudio/blob/3de59e3c7fead55d3792d724847af3e21fadb96a/src/gwt/src/org/rstudio/studio/client/server/remote/RemoteServer.java#L6512

Reply:
I'll bring a fix to Pro myself if confirmed 😄

Reply:
Yes, it looks like an accidental search and replace to me!